poo/intro-POO/fuera-de-clase/archivos/regAutomotor.ts

```diff
@@ -0,0 +1,106 @@
+import * as fs from 'fs';
+import * as ReadlineSync from "readline-sync"; 
+import { strict } from 'assert';
+import { compileFunction } from 'vm';
+import { threadId } from 'worker_threads';
+
+class Auto{
+    private marca: string;
+    private modelo: string;
+    private anio: string;
+    private _patente : string;
+    private kilometraje: string;
+    private _titular: string;
+  
+    constructor(_marca:string,_modelo:string,_anio:string,_patente:string,_km:string,_titular:string){
+        this.marca=_marca;
+        this.modelo=_modelo;
+        this.anio=_anio;
+        this._patente=_patente;
```

Comment:
Por alguna razon empezas los nombres de estas variables con _?

Reply:
Para no confundir los parámetros del constructor con las variables de las propiedades/estados.(La de patente tmb esta con_ en el this. para poder usar el get).